utils/kafka-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaConfig.scala

```diff
@@ -23,15 +24,16 @@ case class KafkaConfig(kafkaAddress: String,
                        // when complex key with its own schema is provided, this flag is false
                        // and all topics related to this config require both key and value schema definitions.
                        useStringForKey: Boolean = true,
-                       schemaRegistryCacheConfig: SchemaRegistryCacheConfig = SchemaRegistryCacheConfig()
+                       schemaRegistryCacheConfig: SchemaRegistryCacheConfig = SchemaRegistryCacheConfig(),
+                       avroPlainTextSerialization: Option[Boolean] = None
```

Comment:
why do we need it? As we already have avroAsJsonSerialization?

Reply:
forgot about second place, it comes from kafkaConfig and we pass it SchemaRegistryClientKafkaConfig